sympy/solvers/solvers.py

```diff
@@ -350,7 +350,11 @@ def check_assumptions(expr, against=None, **assumptions):
         if test is expected:
             continue
         elif test is not None:
-            return False
+            if get is True:
+                return False, key
```

Comment:
Yeah, but I implemented thinking that the user should get the wrong assumptions one by one, so that it becomes easier from him to check. Like in case user provides with a long list of assumptions and all are wrong, it will be a tedious job for him to check, I guess.

Do I need to implement such that all the wrong assumptions gets returned, or will it be okay?